examples/experimental/slpk-via-range-requests/index.html

```diff
@@ -0,0 +1,35 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, height=device-height, initial-scale=1" />
+    <title>Local SLPK visualization</title>
+    <style>
+      html {
+        height: 100%;
+      }
+
+      body {
```

Comment:
Do we need so much styling. The smaller we make the example source files, the better.